docs/rst/user_manual/configuration.rst

```diff
@@ -347,7 +347,9 @@ A complete example of all the configurations described on this page can be found
     # Participant that will communicate with a DDS Router in a different LAN.
     # This Participant will work as the remote DDS Router Client, so it set the connection address of the remote one.
 
-    Wan:                                # Participant Id = simple ; Participant Type = wan
+    Wan:                                # Participant Id = Wan
+
+        type: wan                       # Participant Type = wan
```

Comment:
I prefer it this way, so the user can see an example where a participant name is used as its type
```suggestion
    Wan:                                # Participant Id = Wan
                                            # Participant Type = wan
```

Reply:
:+1:  I added a small comment for readers to notice.